feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go

```diff
@@ -341,7 +341,11 @@ func generateSubIntfPair(t *testing.T, dut *ondatra.DUTDevice, dutPort *ondatra.
 	nextHops := []string{}
 	nextHopCount := 63 // nextHopCount specifies number of nextHop IPs needed.
 	for i := 0; i <= nextHopCount; i++ {
-		vlanID := uint16(i)
+		//yang file using  vlan rang 1-4094 https://github.com/openconfig/public/blob/b34db05e8cf2efe69df3762d4bbd80665e1f9e79/release/models/vlan/openconfig-vlan-types.yang#L133
+		vlanID := uint16(i) + 1
```

Comment:
Should this just be vlanID := uint16(i) ?

Reply:
yes, it has to be. Updated the code